sdk/lib/socks5-listener/src/android.rs

```diff
@@ -47,53 +42,50 @@ pub unsafe extern "C" fn Java_net_nymtech_nyms5_NymProxy_run(
 
     log::debug!("using sp {}", sp_input);
 
-    // TODO pass this callback to blocking_run_client
-    env.call_method(cb_object, "onStart", "()V", &[])
-        .expect("failed to call Java callbacks");
-
     let service_provider = char_p::new(sp_input.as_str());
+
+    let arced = Arc::new(Mutex::new(env));
```

Comment:
You'd probably want to use tokio's async aware mutexes because the std ones could theoretically block the runtime

Reply:
Im not so sure, reading the ref https://docs.rs/tokio/latest/tokio/sync/struct.Mutex.html#which-kind-of-mutex-should-you-use I think using async mutex here does not worth it?
what do you think?